README.md

```diff
@@ -4,30 +4,69 @@ Deblur
 [![Build Status](https://travis-ci.org/biocore/deblur.png?branch=master)](https://travis-ci.org/biocore/deblur)
 [![Coverage Status](https://coveralls.io/repos/github/biocore/deblur/badge.svg?branch=master)](https://coveralls.io/github/biocore/deblur?branch=master)
 
-Deblur is a greedy deconvolution algorithm based on known read error profiles.
-
-Code Development Note
-=====================
-
-Some of the code in the package deblur has been derived from [QIIME](http://qiime.org).
-The contributors to these specific QIIME modules have granted permission
-for this porting to take place and put under the BSD license.
+Deblur is a greedy deconvolution algorithm based on Illumina Miseq/Hiseq error profiles.
 
 Install
 =======
+- Deblur requires Python 3.5. If Python 3.5 is not installed, you can create a conda environment for deblur using:
 
 ```
+conda create -n debenv python=3
+```
+
+and activate it using:
+
+```
+source activate debenv
+```
+
+- Install Deblur dependencies:
+```
 conda install -c biocore VSEARCH MAFFT SortMeRNA numpy
-python setup.py install
 ```
 
+- Install Deblur:
+```
+pip install git+git://github.com/biocore/deblur
```

Comment:
No, we need this to be either:

`pip install deblur`

or 

`conda install deblur` (possibly with `-c biocore`)

The reason is that this install method is not against a specific version but against the repository itself